stetho/src/main/java/com/facebook/stetho/inspector/protocol/module/DOM.java

```diff
@@ -208,6 +216,103 @@ public void run() {
     });
   }
 
+  @ChromeDevtoolsMethod
+  public PerformSearchResponse performSearch(JsonRpcPeer peer, final JSONObject params) {
+    return mDOMProvider.postAndWait(new UncheckedCallable<PerformSearchResponse>() {
+      @Override
+      public PerformSearchResponse call() {
+        final PerformSearchRequest request = mObjectMapper.convertValue(
+            params,
+            PerformSearchRequest.class);
+
+        // Searches are case insensitive so convert
+        // the query to lower case for simpler comparisons.
+        final String lowerCaseQuery = request.query.toLowerCase();
+
+        final Object root = mShadowDOM.getRootElement();
+
+        // Each search action has a unique ID so that
+        // it can be queried later.
+        final String id = UUID.randomUUID().toString();
```

Comment:
Strongly prefer monotonically increasing identifiers.  This isn't a security implication and therefore the extra cost and complexity of a UUID is not warranted.